Rmarkdown/Adverse_Events.Rmd

```diff
@@ -46,6 +38,7 @@ my.data <- read.csv("../Data/AE_xgx.csv")
 my.data$DOSE_label <- paste(my.data$Dose,"mg")
```

Comment:
maybe should rename "my.data" as "ae_data"

Reply:
I am going to keep it my.data to be consistent with other pages

Reply:
I did it, we can take care of consistency later...